src/utils/polyfill.js

```diff
@@ -0,0 +1,8 @@
+import { Platform } from 'react-native'
+
+// noinspection JSConstantReassignment
```

Comment:
what's this here for?

Reply:
This is for H3, it needs a shim to work but it doesnt actually use the document so it works great. I added that noinspect to ignore some lint error.